youtube_dl/extractor/googledrive.py

```diff
@@ -0,0 +1,106 @@
+from .common import InfoExtractor
+from ..utils import RegexNotFoundError
+
+class GoogleDriveIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)?(?:video\.google\.com/get_player\?.*?docid=|(?:docs|drive)\.google\.com/(?:uc\?.*?id=|file/d/))(?P<id>.+?)(?:&|/|$)'
+    _TEST = {
+        'url': 'https://drive.google.com/file/d/0BzpExh0WzJF0NlR5WUlxdEVsY0U/edit?pli=1',
```

Comment:
This is [clearly a copyrighted video](https://en.wikipedia.org/wiki/Fairy_Tail#Anime), and I suspect that they don't have the rights for hosting it. Please use a video uploaded with author permission.